modules/internal/String.chpl

```diff
@@ -1906,12 +1903,10 @@ module String {
                 character converted to uppercase, and all cased characters
                 following another cased character converted to lowercase.
 
-      Note: At present, this performs a one-to-one character mapping,
-      which is all that the underlying C library supports.  In the future,
-      it may be desirable to explore supporting special-case one-to-many
-      Unicode mappings that may change the length of the string.
-      Since Unicode changes and we would rather not track all the changes
-      ourselves, this would involve adopting an external library.
+      .. note::
+        
+        The case change operation is not performed on characters whose cases
```

Comment:
Can we change this, and the other instance of it, to say "is not currently performed"? I don't think this is the long-term situation, just an oddity/bug in the current implememtiation.